gamefixes/242760.py

```diff
@@ -0,0 +1,50 @@
+""" Game fix for The Forest
+"""
+
+# pylint: disable=C0103
+
+import os
+
+from protonfixes import util
+from protonfixes.logger import log
+
+
+def check_smt() -> bool:
```

Comment:
I thought about it, as well as adding a function to set the topology.
Some other gamefixes limit the cpu cores, because some engines break with high counts.
This may be another pull request though.

@GloriousEggroll What do you think?